docs/docs/quickstart.md

```diff
@@ -237,9 +237,9 @@ The output from the command above should display a list of any matching emails.
 of the email you would like to use for testing restore.
 
 ```text
-  ID            Sender                 Subject                                  Received
-  360bf6840396  phish@contoso.info     Re: Request for Apple/Amazon gift cards  2022-10-18T02:27:47Z
-  84dbad89b9f5  ravi@cohovineyard.com  Come join us!                            2022-10-19T06:12:08Z
+  ID            Sender                 Subject                                  Received              Created               Modified
```

Comment:
I don't think we should be displaying these fields (at least not be default) since it increases the chances of lines wrapping and the created and modified, while useful in the manifest, are not that meaningful to an end user to warrant sacrificing the horizontal screen real estate.

We can file a separate enhancement that would be for a 'wide' display option

Reply:
Makes sense. I'll drop showing created/modified field for exchange items.

Reply:
Without the Created and Modified, there is no need for this PR. I'll close this as well.